src/infra/dev/api/oi.tpl.json

```diff
@@ -289,29 +289,634 @@
+                  },
+                  "organization": {
+                    "type": "string",
+                    "example": "https://organization.com"
+                  },
+                  "description": {
+                    "type": "string",
+                    "example": "Registered via OpenID Connect Dynamic Registration"
+                  },
+                  "environment": {
+                    "type": "string",
+                    "example": "ALL"
+                  },
+                  "master": {
+                    "type": "boolean",
+                    "example": false
+                  },
+                  "spid_requested_attributes": {
+                    "type": "array",
+                    "items": {
+                      "type": "string"
+                    },
+                    "description": "Custom parameter which return all SPID attributes selected, https://docs.italia.it/italia/spid/spid-regole-tecniche/it/stabile/attributi.html",
+                    "example": ["spidCode", "name", "familyName", "fiscalNumber"]
+                  }
+                }
+              }
+            }
+          },
+          "description": "Request body",
+          "required": true
+        },
+        "responses": {
+          "201": {
+            "description": "Successful registration including details about the registered values.",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "type": "object",
+                  "properties": {
+                    "client_id": {
+                      "type": "string",
+                      "example": "47231c87-74f3-4990-99ad-31e8a801e3b5",
+                      "description": "The registered client_id"
+                    },
+                    "client_secret_expires_at": {
+                      "type": "integer",
+                      "example": 0,
+                      "description": "The timestamp in seconds indicating the time of expiration for this value. 0 indicates no expiration"
+                    },
+                    "client_secret": {
+                      "type": "string",
+                      "example": "e4cccd81-1d19-41e7-b4b0-488aa28af77b",
+                      "description": "The registered client_secret"
+                    },
+                    "registration_access_token": {
+                      "type": "string",
+                      "example": "db4b7554-e359-43ed-bbf6-1a50909ceeef",
+                      "description": "An access_token to be used at {registration_client_uri} to retrieve the registered information"
+                    },
+                    "registration_client_uri": {
+                      "type": "string",
+                      "example": "https://server.example.org/connect/register?client_id=s6BhdRkqt3",
+                      "description": "The URI to retrieved the registered values"
+                    },
```

Comment:
given that this is implemented using the path parameter, this should be deleted, right?

Reply:
my bad, going to fix it